wgsl/index.bs

```diff
@@ -12689,6 +12689,13 @@ WGSL does not specify a [=ieee754/rounding mode=].
 Note: Floating point types include positive and negative infinity, so
 the correctly rounded result may be finite or infinite.
 
+Note: The result of an operation when computed with unbounded precision may require precision
+which exceeds that of a double. An example of such a case is in `x - y` where `x=1.0`
+and `y=1.17e-38` (smallest positive normal single precision float). These numbers have
+exponents are that are `126` units apart. Double precision only has `52` explicit bits of mantissa,
```

Comment:
Replace "Double precision..." with "The [[!IEEE-754|IEEE-754]] [=ieee754/binary64=] (double precision) format only has 52 bits in its significand,"

(We use "significand" instead of mantissa...)